Fandom Creator/JavaScript.md

```diff
@@ -34,6 +36,30 @@ class Toggle extends React.Component {
 }
 ```
 
+## Instance fields in a class
+Similar to the public class instance methods, instance fields do not need to be set in the constructor. All instance fields should be initialized in the root of the class. Fields should _always_ be declared as "instance fields" over setting them in the constructor (even when a constructor is required for other reasons). 
+
+Babel ultimately transpiles them _nearly_ the same code (https://babeljs.io/docs/en/babel-plugin-proposal-class-properties) Which is a current stage 3 proposal to add to javascript (https://github.com/tc39/proposal-class-fields)  
```

Comment:
Is there a good reason for this or is it your personal preference?

Reply:
A couple of reasons:
* It's very common among react projects
* It's consistent with RDS
* The source code is shorter (not `this` and no `super(props)`

Reply:
Could you add some of these reasons to the guidelines? Will be helpful for future us to remember why we made the guideline in the first place.

Reply:
Ditto.
Also, glad to see the Babel breakdown- for new(ish) folks, it's good, methinks, to make the distinction, esp. in a case, such as this one, that Babel is doing the work, and that it isn't ES6 syntax.